File: app/src/main/java/social/entourage/android/invite/InviteModule.java
package social.entourage.android.invite;

import dagger.Module;
import dagger.Provides;

/**
 * Created by mihaiionescu on 12/07/16.
 */
@Module
public class InviteModule {
    private final InviteBaseFragment fragment;

    public InviteModule(InviteBaseFragment fragment) {
        this.fragment = fragment;
    }

    @Provides
    public InviteBaseFragment providesInviteContactsFragment() {
        return fragment;
    }
}

File: app/src/main/java/social/entourage/android/entourage/invite/InviteModule.kt
package social.entourage.android.entourage.invite

import dagger.Module
import dagger.Provides

/**
 * Created by mihaiionescu on 12/07/16.
 */
@Module
class InviteModule(private val fragment: InviteBaseFragment) {
    @Provides
    fun providesInviteContactsFragment(): InviteBaseFragment {
        return fragment
    }

}